frontend/src/components/QcControls.vue

```diff
@@ -1,19 +1,70 @@
 <script setup>
-    import ClaimWidget from "@/components/ClaimWidget.vue";
-    import QcWidget from "@/components/QcWidget.vue";
+import { computed, defineProps, ref, watch } from "vue";
+import ClaimWidget from "@/components/ClaimWidget.vue";
+import QcWidget from "@/components/QcWidget.vue";
+import { useWellStore } from "@/stores/focusWell.js";
 
-    const emit = defineEmits(['wellChanged']);
+const emit = defineEmits(['wellChanged']);
+const focusWell = useWellStore();
 
-    function changeTab() {
-        emit('wellChanged', 'in_progress');
+const props = defineProps({
+    user: String, // the user for this session
+});
+
+function changeTab() {
+    emit('wellChanged', 'in_progress');
+}
+
+let override = ref(false); // Override interface restrictions on modifying QC states
+function flipOverride() {
+    override.value = !override.value;
+}
+watch(focusWell, () => { override.value = false }); // Turn override off if the well changes
+
+const owner = computed(() => {
+    if (focusWell.getAssessor && props.user && focusWell.getAssessor == props.user) {
+        return true
+    } else {
+        return false
+    }
```

Comment:
Why is it fine to return `false` here, while in other cases `null` is used?

Reply:
It can be properly boolean because it's not used in a template render where false leads to `disabled=""` which is `true`.